scripts/check_go_version.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+# check_go_version.sh - Check current go version against minimal required
+#
+# Syntax:   check_go_version.sh
+
+GOMINVERSION_MAJOR=1
+GOMINVERSION_MINOR=14
+
+UNAME=$(uname)
+if [[ "${UNAME}" == *"MINGW"* ]]; then
```

Comment:
Why are we concerned with GOPATH at all? It won't affect what `go` is found on $PATH

Reply:
fixed